PushApp.ts

```diff
@@ -106,5 +105,14 @@ export class PushApp extends App {
             i18nLabel: 'config_push_media_flow',
         });
 
+        await configuration.settings.provideSetting({
+            id:  REQUEST_TIMEOUT,
+            type: SettingType.NUMBER,
+            packageValue: 30,
+            required: true,
+            public: false,
+            i18nLabel: 'config_push_timeout',
```

Comment:
Since this config is for both RocketChat and Push, you should rename the i18n label to "config_timeout".